front/src/modules/trainschedule/components/ManageTrainSchedule/TrainSettings.tsx

```diff
@@ -104,7 +83,9 @@ export default function TrainSettings() {
             </>
           }
           id="timetable-item-name"
-          onChange={(e: React.ChangeEvent<HTMLInputElement>) => setName(e.target.value)}
+          onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
```

Comment:
nit : while at it, we could also change that :) Typescript infers automatically the type of the event (same for the 2 other fields)
```suggestion
          onChange={(e) =>
```

Reply:
Thanks for the feedback. I"ll make the changes Wedesday night :)

Reply:
gonna have to wait until Firday.... too much going now this week.

Reply:
@SharglutDev Thanks for the catch :) Fixed.